spec/packs_spec.rb

```diff
@@ -306,31 +307,34 @@ def write_codeownership_config
         EXPECTED
       end
 
-      it 'adds a README_TODO.md file as a placeholder' do
+      it 'adds a README.md file as a placeholder' do
         Packs.create_pack!(pack_name: 'packs/organisms')
         ParsePackwerk.bust_cache!
-        actual_readme_todo = ParsePackwerk.find('packs/organisms').directory.join('README_TODO.md')
-        expect(actual_readme_todo.read).to eq expected_readme_todo
+        actual_readme = ParsePackwerk.find('packs/organisms').directory.join('README.md')
+        expect(actual_readme.read).to eq expected_readme
       end
 
       context 'app has one pack with an outdated README_TODO.md' do
-        it 'overwrites the README_TODO.md' do
+        it 'deletes the README_TODO.md and adds a README' do
           write_file('packs/organisms/README_TODO.md', 'This is outdated')
           write_package_yml('packs/organisms')
-          actual_readme_todo = ParsePackwerk.find('packs/organisms').directory.join('README_TODO.md')
-          expect(actual_readme_todo.read).to eq 'This is outdated'
+          readme_todo = ParsePackwerk.find('packs/organisms').directory.join('README_TODO.md')
+          expect(readme_todo.read).to eq 'This is outdated'
+
           Packs.create_pack!(pack_name: 'packs/organisms')
-          expect(actual_readme_todo.read).to eq expected_readme_todo
+          actual_readme = ParsePackwerk.find('packs/organisms').directory.join('README.md')
+          expect(actual_readme.read).to eq expected_readme
+          expect(readme_todo.exist?).to eq false
         end
       end
 
       context 'app has one pack with a README.md' do
-        it 'does not add a README_TODO.md file' do
+        it 'does not change the README.md file' do
           write_package_yml('packs/organisms')
-          write_file('packs/organisms/README.md')
-          actual_readme_todo = ParsePackwerk.find('packs/organisms').directory.join('README_TODO.md')
+          write_file('packs/organisms/README.md', 'This is the original README.md')
+          actual_readme = ParsePackwerk.find('packs/organisms').directory.join('README.md')
           Packs.create_pack!(pack_name: 'packs/organisms')
-          expect(actual_readme_todo.exist?).to eq false
+          expect(actual_readme.read).to eq 'This is the original README.md'
         end
       end
```

Comment:
Looks like the spec on 347/343 should be updated too but I can't comment down there